hooks/post-command

```diff
@@ -11,13 +11,30 @@ if [[ "${#CORE_DUMPS[@]}" > 0 ]]; then
     echo "--- List of core dumps"
     ls -lah "${CORE_DUMPS[@]}"
 
-    # Start compressing/collecting corefiles in the background
+    # Start a subshell for compressing/collecting corefiles in the background in parallel
+    (
     PIDS=()
     for COREFILE in "${CORE_DUMPS[@]}"; do
         compress_bundle "${COREFILE}" &
         PIDS+=( "$!" )
     done
 
+    # Wait for compression to finish
+    echo "--- Waiting upon compression/bundling"
+    for PID in "${PIDS[@]}"; do
+        wait "${PID}"
+    done
```

Comment:
Should we change this to wait for all of them regardless of status, instead of waiting for the first one that fails, to be more robust here?
```suggestion
    wait -f "${PIDS[@]}"
```